keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -2401,7 +2401,13 @@ in back-to-back releases.
 - support the list of features mentioned under phrase 2
 
 ## GA
-- complete type check for CRD and aggregated types
+- Complete type check for CRD and aggregated types
+- Scalability evaluation. Evaluate ValidatingAdmissionPolicy scalability including how many validators it could run, runtime cost evaluation, evaluating the reasonable ResourceQuota for
+  ValidatingAdmissionPolicy, how much faster comparing with Webhook, the scale target, etc.
+- Fix the known issues along the way([variables should keep the type info](https://github.com/kubernetes/kubernetes/issues/120239), [variables should mark omitempty](https://github.com/kubernetes/kubernetes/issues/120491), [properly escape](https://github.com/kubernetes/kubernetes/issues/118230))
+- Define the mechanism for CEL library update policy after GA(version skew policy, the policy to update CEL library, etc). Sync with Declarative Validation on CEL libraries to prevent unexpected libraries be added after GA.
```

Comment:
perhaps mock it up with an isIP function?

Reply:
TheIP/CIDR cel library is going to be added in https://github.com/kubernetes/kubernetes/pull/121912. We could have a readme added as a guidance for add/delete/modify cel library later. How does this sound? :)

Reply:
> Define the mechanism for CEL library update policy after GA(version skew policy

We introduced [Compatibility Version and introduced versions](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/cel/environment/base.go#L45) as the mechanism.

+1 to providing documented guidance on library changes, either in a readme or somewhere in https://github.com/kubernetes/community/tree/master/contributors/devel